Update select_elements.en.md

insumanth opened the pull request:
Replaced the `&lt;` and `&gt;` to `<` and `>` in a example for python and ruby.

### Description
The `&lt;` and `&gt;` are not rendered to `<` and `>` symbols in the web page as they are formatted as code.
It is displayed as `&lt;select&gt;` in the web page, so made this changes so it will be displayed as `<select>`.

### Motivation and Context
Please add this proposed changes as it may affect people who are reading examples.

### Types of changes
- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [x] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist
- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

insumanth committed:
Update select_elements.en.md

Replaced the &lt; and &gt; to < and > in a example for python and ruby.
The &lt; and &gt; are not rendered to < and > symbols as they are formatted as code.
It is displayed as '&lt;select&gt;' in the webpage, so made this changes so it will be displayed as '<select>'. 
Please add this proposed changes as it may affect people who are reading examples.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/SeleniumHQ/seleniumhq.github.io?pullRequest=880) <br/>All committers have signed the CLA.

diemol reviewed (changes requested):
Could you please change this also in the `select_elements.ja.md`, `select_elements.pt-br.md` and `select_elements.zh-cn.md`?

insumanth commented:
Ok. I'll make these changes also

insumanth committed:
Update select_elements.ja.md

Replaced the &lt; and &gt; to < and > in a example for python and ruby.
The &lt; and &gt; are not rendered to < and > symbols as they are formatted as code.
It is displayed as '&lt;select&gt;' in the webpage, so made this changes so it will be displayed as '<select>'. 
Please add this proposed changes as it may affect people who are reading examples.

insumanth committed:
Update select_elements.pt-br.md

Replaced the &lt; and &gt; to < and > in a example for python and ruby.
The &lt; and &gt; are not rendered to < and > symbols as they are formatted as code.
It is displayed as '&lt;select&gt;' in the webpage, so made this changes so it will be displayed as '<select>'. 
Please add this proposed changes as it may affect people who are reading examples.

insumanth committed:
Update select_elements.zh-cn.md

Replaced the &lt; and &gt; to < and > in a example for python and ruby.
The &lt; and &gt; are not rendered to < and > symbols as they are formatted as code.
It is displayed as '&lt;select&gt;' in the webpage, so made this changes so it will be displayed as '<select>'. 
Please add this proposed changes as it may affect people who are reading examples.

insumanth commented:
Hi @diemol, I updated the changes in other mentioned files also.
Please review and merge. Thank you.

diemol commented:
Could you please fix the merge conflicts?

harsha509 commented:
Hi @insumanth,

Hope you're doing well.

Did you get a chance to look into the above comments?

titusfortner commented:
Since I'm the one who made it complicated, I cherry picked/squashed/merged for you: https://github.com/SeleniumHQ/seleniumhq.github.io/commit/c5e7d7dc3ec0de1b25a94ad13d3240ed3795684c

Thanks for your contribution!
